discord/ext/events/errors.py

```diff
@@ -0,0 +1,5 @@
+class EventsException(Exception):
```

Comment:
This - and all other extension-level exceptions - should at the very least derive from [DiscordException](https://github.com/Rapptz/discord.py/blob/master/discord/errors.py#L27)